packages/react-native-web/src/vendor/react-native/JSEventLoopWatchdog/index.js

```diff
@@ -39,7 +38,7 @@ const JSEventLoopWatchdog = {
     totalStallTime = 0;
     stallCount = 0;
     longestStall = 0;
-    lastInterval = performanceNow();
+    lastInterval = global.performance.now();
```

Comment:
I'm not sure if `global` is reliable on web vs `window`.

Reply:
I've replaced with `window` now. Thanks for catching that